Rakefile

```diff
@@ -6,10 +6,27 @@ require 'hoe'
 
 Hoe.plugin :bundler
 Hoe.plugin :doofus
-Hoe.plugin :email unless ENV['CI'] or ENV['TRAVIS']
 Hoe.plugin :gemspec2
 Hoe.plugin :git
-Hoe.plugin :travis
+
+if RUBY_VERSION < '1.9'
+  class Array
+    def to_h
+      Hash[*self.flatten(1)]
+    end
+  end
+
+  class Gem::Specification
```

Comment:
Supported in a later version of RubyGems than is installed on the Docker container I tested with.